diff --git a/node_modules/preact/compat/src/index.d.ts b/node_modules/preact/compat/src/index.d.ts
index 3a054da..434b6a3 100644
--- a/node_modules/preact/compat/src/index.d.ts
+++ b/node_modules/preact/compat/src/index.d.ts
@@ -217,4 +217,9 @@ declare namespace React {
 		callback: () => void
 	): void;
 	export const unstable_now: () => number;
+
+  // Custom
+  export type ComponentPropsWithoutRef<
+  	C extends React.ComponentType<any> | keyof JSXInternal.IntrinsicElements,
+  > = React.PropsWithoutRef<ComponentProps<C>>;
 }
