diff --git a/compat/src/index.d.ts b/compat/src/index.d.ts
index dbbc61b95498ee2e83e2fc11d31ee376ed584521..87579b4043a86b05e352a154f1aedd0a2074c3c0 100644
--- a/compat/src/index.d.ts
+++ b/compat/src/index.d.ts
@@ -217,4 +217,9 @@ declare namespace React {
 		callback: () => void
 	): void;
 	export const unstable_now: () => number;
+
+  // Custom
+  export type ComponentPropsWithoutRef<
+  	C extends React.ComponentType<any> | keyof JSXInternal.IntrinsicElements,
+  > = React.PropsWithoutRef<ComponentProps<C>>;
 }
