diff --git a/node_modules/preact/compat/src/index.d.ts b/node_modules/preact/compat/src/index.d.ts
index 6cbf585..ffef881 100644
--- a/node_modules/preact/compat/src/index.d.ts
+++ b/node_modules/preact/compat/src/index.d.ts
@@ -54,6 +54,7 @@ declare namespace React {
 	export import cloneElement = preact.cloneElement;
 	export import ComponentProps = preact.ComponentProps;
 	export import ReactNode = preact.ComponentChild;
+	export import ReactElement = preact.VNode;
 
 	// Suspense
 	export import Suspense = _Suspense.Suspense;
@@ -205,4 +206,9 @@ declare namespace React {
 		callback: () => void
 	): void;
 	export const unstable_now: () => number;
+
+  // Custom
+  export type ComponentPropsWithoutRef<
+    C extends React.ComponentType<any> | keyof JSXInternal.IntrinsicElements,
+  > = React.PropsWithoutRef<ComponentProps<C>>;
 }
