diff --git a/compat/src/index.d.ts b/compat/src/index.d.ts
index 3a054da47c464adf3a93d58938fbf33ef6b162d3..434b6a3b7bc8534987053a0c2a6f39bffc0e9af6 100644
--- a/compat/src/index.d.ts
+++ b/compat/src/index.d.ts
@@ -217,4 +217,9 @@ declare namespace React {
 		callback: () => void
 	): void;
 	export const unstable_now: () => number;
+
+  // Custom
+  export type ComponentPropsWithoutRef<
+  	C extends React.ComponentType<any> | keyof JSXInternal.IntrinsicElements,
+  > = React.PropsWithoutRef<ComponentProps<C>>;
 }
