diff --git a/node_modules/wxt/dist/cli.js b/node_modules/wxt/dist/cli.js
index 7521b9c..b15c476 100644
--- a/node_modules/wxt/dist/cli.js
+++ b/node_modules/wxt/dist/cli.js
@@ -2174,7 +2174,7 @@ function hashContentScriptOptions(options) {
   return JSON.stringify(
     Object.entries(withDefaults).map(([key, value]) => {
       if (Array.isArray(value))
-        return [key, value.sort()];
+        return [key, value.toSorted()];
       else
         return [key, value];
     }).sort((l, r) => l[0].localeCompare(r[0]))
