diff --git a/dist/chunk-ZONHOHIX.js b/dist/chunk-P6KYES6W.js
similarity index 99%
rename from dist/chunk-ZONHOHIX.js
rename to dist/chunk-P6KYES6W.js
index 4c4d3fc2249a79353e8120617b402f90f7c1dae4..6a3a6e632f718c4813ea0128bd96005c27ffd5e2 100644
--- a/dist/chunk-ZONHOHIX.js
+++ b/dist/chunk-P6KYES6W.js
@@ -16,7 +16,7 @@ import {
 } from "./chunk-QGM4M3NI.js";
 
 // package.json
-var version = "0.18.13";
+var version = "0.18.14";
 
 // src/core/utils/paths.ts
 import systemPath from "node:path";
diff --git a/dist/cli.js b/dist/cli.js
index 16788e48914d958bde3534c5954474931e502d6f..4af0f24b514bb58af45f5c80d93bcb2865d14f65 100644
--- a/dist/cli.js
+++ b/dist/cli.js
@@ -2788,7 +2788,7 @@ function getChunkSortWeight(filename) {
 import pc4 from "picocolors";
 
 // package.json
-var version = "0.18.13";
+var version = "0.18.14";
 
 // src/core/utils/log/printHeader.ts
 function printHeader() {
@@ -4186,13 +4186,15 @@ async function zip(config) {
 }
 async function zipDir(directory, outputPath, options) {
   const archive = new JSZip();
-  const files = (await glob7("**/*", {
+  const files = (await glob7([
+    "**/*",
+    /* workaround for https://github.com/wxt-dev/wxt/issues/604 */
+    ...options?.include || []
+  ], {
     cwd: directory,
     // Ignore node_modules, otherwise this glob step takes forever
     ignore: ["**/node_modules"],
     onlyFiles: true
-    // TODO: Fix #738
-    // dot: true,
   })).filter((relativePath) => {
     return options?.include?.some((pattern) => minimatch2(relativePath, pattern)) || !options?.exclude?.some((pattern) => minimatch2(relativePath, pattern));
   });
diff --git a/dist/index.cjs b/dist/index.cjs
index d9badf8e324861d847adab2264396a68e3490399..24ee3f4a3298d4a990d946e9f905a68414ade266 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -12970,7 +12970,7 @@ function getChunkSortWeight(filename) {
 var import_picocolors4 = __toESM(require("picocolors"), 1);
 
 // package.json
-var version = "0.18.13";
+var version = "0.18.14";
 
 // src/core/utils/building/internal-build.ts
 var import_fast_glob5 = __toESM(require("fast-glob"), 1);
@@ -14372,13 +14372,15 @@ async function zip(config) {
 }
 async function zipDir(directory, outputPath, options) {
   const archive = new import_jszip.default();
-  const files = (await (0, import_fast_glob7.default)("**/*", {
+  const files = (await (0, import_fast_glob7.default)([
+    "**/*",
+    /* workaround for https://github.com/wxt-dev/wxt/issues/604 */
+    ...options?.include || []
+  ], {
     cwd: directory,
     // Ignore node_modules, otherwise this glob step takes forever
     ignore: ["**/node_modules"],
     onlyFiles: true
-    // TODO: Fix #738
-    // dot: true,
   })).filter((relativePath) => {
     return options?.include?.some((pattern) => (0, import_minimatch2.minimatch)(relativePath, pattern)) || !options?.exclude?.some((pattern) => (0, import_minimatch2.minimatch)(relativePath, pattern));
   });
diff --git a/dist/index.d.cts b/dist/index.d.cts
index b64172df3ab7b99efbd33562185a637911fc885f..febded4ae5af4555b6c045aaa63e69479b13cebf 100644
--- a/dist/index.d.cts
+++ b/dist/index.d.cts
@@ -76,6 +76,6 @@ declare function prepare(config: InlineConfig): Promise<void>;
  */
 declare function zip(config?: InlineConfig): Promise<string[]>;
 
-var version = "0.18.13";
+var version = "0.18.14";
 
 export { BuildOutput, ExtensionRunnerConfig, InlineConfig, UserConfig, WxtDevServer, build, clean, createServer, defineConfig, defineRunnerConfig, initialize, prepare, version, zip };
diff --git a/dist/index.d.ts b/dist/index.d.ts
index 4b7d76d2149fa8e1e69e66ec4413ffce90bf628d..8c9dc53b92e980ffa3775cd88281ec025d3958a1 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -76,6 +76,6 @@ declare function prepare(config: InlineConfig): Promise<void>;
  */
 declare function zip(config?: InlineConfig): Promise<string[]>;
 
-var version = "0.18.13";
+var version = "0.18.14";
 
 export { BuildOutput, ExtensionRunnerConfig, InlineConfig, UserConfig, WxtDevServer, build, clean, createServer, defineConfig, defineRunnerConfig, initialize, prepare, version, zip };
diff --git a/dist/index.js b/dist/index.js
index 4437530acae8695513263ec5d1eddd89592657a7..3d2c5217591e5b87607ca33f5f51d8a8237f32d9 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -19,7 +19,7 @@ import {
   unnormalizePath,
   version,
   wxt
-} from "./chunk-ZONHOHIX.js";
+} from "./chunk-P6KYES6W.js";
 import "./chunk-BERPNPEZ.js";
 import "./chunk-6XSIWUWF.js";
 import {
@@ -605,13 +605,15 @@ async function zip(config) {
 }
 async function zipDir(directory, outputPath, options) {
   const archive = new JSZip();
-  const files = (await glob2("**/*", {
+  const files = (await glob2([
+    "**/*",
+    /* workaround for https://github.com/wxt-dev/wxt/issues/604 */
+    ...options?.include || []
+  ], {
     cwd: directory,
     // Ignore node_modules, otherwise this glob step takes forever
     ignore: ["**/node_modules"],
     onlyFiles: true
-    // TODO: Fix #738
-    // dot: true,
   })).filter((relativePath) => {
     return options?.include?.some((pattern) => minimatch(relativePath, pattern)) || !options?.exclude?.some((pattern) => minimatch(relativePath, pattern));
   });
diff --git a/dist/testing.js b/dist/testing.js
index 79f65df708a348033b8bf7dc85c237cc360530ed..410e931cfd5382f285b51b0303f983259638c525 100644
--- a/dist/testing.js
+++ b/dist/testing.js
@@ -6,7 +6,7 @@ import {
   tsconfigPaths,
   vitePlugin,
   webextensionPolyfillMock
-} from "./chunk-ZONHOHIX.js";
+} from "./chunk-P6KYES6W.js";
 import "./chunk-BERPNPEZ.js";
 import "./chunk-6XSIWUWF.js";
 import "./chunk-BM6QYGAW.js";
