diff --git a/dist/chunk-EI323YQS.js b/dist/chunk-V27ZDDHA.js
similarity index 99%
rename from dist/chunk-EI323YQS.js
rename to dist/chunk-V27ZDDHA.js
index cca6f0ede4afbb3a06fefb66cc5a7be1f1a68760..9f133dd568081cefbd9f8a76ec9f5469488687ba 100644
--- a/dist/chunk-EI323YQS.js
+++ b/dist/chunk-V27ZDDHA.js
@@ -12,7 +12,7 @@ import {
 } from "./chunk-VBXJIVYU.js";
 
 // package.json
-var version = "0.18.0";
+var version = "0.18.1";
 
 // src/core/utils/paths.ts
 import systemPath from "node:path";
diff --git a/dist/cli.js b/dist/cli.js
index a8b21768a09840a82265310cda480b6aafcc3df8..76cf9f0ae6c6ac8a6feb1a64ed74d5fc680b5fca 100644
--- a/dist/cli.js
+++ b/dist/cli.js
@@ -2515,7 +2515,7 @@ function getChunkSortWeight(filename) {
 import pc4 from "picocolors";
 
 // package.json
-var version = "0.18.0";
+var version = "0.18.1";
 
 // src/core/utils/log/printHeader.ts
 function printHeader() {
@@ -3921,7 +3921,11 @@ async function zip(config) {
 }
 async function zipDir(directory, outputPath, options) {
   const archive = new JSZip();
-  const files = (await glob5("**/*", {
+  const files = (await glob5([
+    "**/*",
+    /* workaround for https://github.com/wxt-dev/wxt/issues/604 */
+    ...options?.include || []
+  ], {
     cwd: directory,
     // Ignore node_modules, otherwise this glob step takes forever
     ignore: ["**/node_modules"],
diff --git a/dist/index.cjs b/dist/index.cjs
index f79adbfbe9289c883838944d4be534408f8da9eb..91c7e0d9cea64c8f277ccb07a69d73d5644fcb09 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -6805,7 +6805,7 @@ function getChunkSortWeight(filename) {
 var import_picocolors4 = __toESM(require("picocolors"), 1);
 
 // package.json
-var version = "0.18.0";
+var version = "0.18.1";
 
 // src/core/utils/building/internal-build.ts
 var import_fast_glob3 = __toESM(require("fast-glob"), 1);
@@ -8215,7 +8215,11 @@ async function zip(config) {
 }
 async function zipDir(directory, outputPath, options) {
   const archive = new import_jszip.default();
-  const files = (await (0, import_fast_glob5.default)("**/*", {
+  const files = (await (0, import_fast_glob5.default)([
+    "**/*",
+    /* workaround for https://github.com/wxt-dev/wxt/issues/604 */
+    ...options?.include || []
+  ], {
     cwd: directory,
     // Ignore node_modules, otherwise this glob step takes forever
     ignore: ["**/node_modules"],
diff --git a/dist/index.d.cts b/dist/index.d.cts
index 97dcdf08c31822a00c84705d035b237bd43e7ca5..30cce3df2f5605f195ea4c3b0824c777f8599428 100644
--- a/dist/index.d.cts
+++ b/dist/index.d.cts
@@ -65,6 +65,6 @@ declare function prepare(config: InlineConfig): Promise<void>;
  */
 declare function zip(config?: InlineConfig): Promise<string[]>;
 
-var version = "0.18.0";
+var version = "0.18.1";
 
 export { BuildOutput, ExtensionRunnerConfig, InlineConfig, UserConfig, WxtDevServer, build, clean, createServer, defineConfig, defineRunnerConfig, initialize, prepare, version, zip };
diff --git a/dist/index.d.ts b/dist/index.d.ts
index 08e910cc3fcbdf27503ffc368489f00fc82692a4..0ff4552bc168bf8214438b2266bb7d634f082fdc 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -65,6 +65,6 @@ declare function prepare(config: InlineConfig): Promise<void>;
  */
 declare function zip(config?: InlineConfig): Promise<string[]>;
 
-var version = "0.18.0";
+var version = "0.18.1";
 
 export { BuildOutput, ExtensionRunnerConfig, InlineConfig, UserConfig, WxtDevServer, build, clean, createServer, defineConfig, defineRunnerConfig, initialize, prepare, version, zip };
diff --git a/dist/index.js b/dist/index.js
index a901ceb32740ebfd650ae06b12bb5ecde1e3da56..dc6eea938ebbef78d8e94415a51ad89211272881 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -19,7 +19,7 @@ import {
   unnormalizePath,
   version,
   wxt
-} from "./chunk-EI323YQS.js";
+} from "./chunk-V27ZDDHA.js";
 import "./chunk-5X3S6AWF.js";
 import {
   consola
@@ -609,7 +609,11 @@ async function zip(config) {
 }
 async function zipDir(directory, outputPath, options) {
   const archive = new JSZip();
-  const files = (await glob2("**/*", {
+  const files = (await glob2([
+    "**/*",
+    /* workaround for https://github.com/wxt-dev/wxt/issues/604 */
+    ...options?.include || []
+  ], {
     cwd: directory,
     // Ignore node_modules, otherwise this glob step takes forever
     ignore: ["**/node_modules"],
diff --git a/dist/testing.js b/dist/testing.js
index c25cc30b048ee887fb8449802a33eda844950c50..4e7fdee772267e5c0658caadff7934c4e46ca54f 100644
--- a/dist/testing.js
+++ b/dist/testing.js
@@ -5,7 +5,7 @@ import {
   tsconfigPaths,
   unimport,
   webextensionPolyfillMock
-} from "./chunk-EI323YQS.js";
+} from "./chunk-V27ZDDHA.js";
 import "./chunk-5X3S6AWF.js";
 import "./chunk-ZZCTFNQ5.js";
 import "./chunk-VBXJIVYU.js";
