diff --git a/dist/cli.js b/dist/cli.js
index cb2c7a721fa378f117e16f8fe53a644f5bdee8ef..3621a665cf9e2e94ccdba310cc58ffa828259a88 100644
--- a/dist/cli.js
+++ b/dist/cli.js
@@ -3822,7 +3822,11 @@ async function zip(config) {
 }
 async function zipDir(directory, outputPath, options) {
   const archive = new JSZip();
-  const files = (await glob5("**/*", {
+  const files = (await glob5([
+    "**/*",
+    /* workaround for https://github.com/wxt-dev/wxt/issues/604 */
+    ...options?.include || []
+  ], {
     cwd: directory,
     // Ignore node_modules, otherwise this glob step takes forever
     ignore: ["**/node_modules"],
diff --git a/dist/index.cjs b/dist/index.cjs
index 3d03c0ad464cbf1f1b36c6639802c8eacbaaada6..3e1482d21af46990822dc01847599479a0def49a 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -6244,7 +6244,11 @@ async function zip(config) {
 }
 async function zipDir(directory, outputPath, options) {
   const archive = new import_jszip.default();
-  const files = (await (0, import_fast_glob5.default)("**/*", {
+  const files = (await (0, import_fast_glob5.default)([
+    "**/*",
+    /* workaround for https://github.com/wxt-dev/wxt/issues/604 */
+    ...options?.include || []
+  ], {
     cwd: directory,
     // Ignore node_modules, otherwise this glob step takes forever
     ignore: ["**/node_modules"],
diff --git a/dist/index.js b/dist/index.js
index 3c1c62902c9e50dae801c2c2535e05080037074c..29f91beb32a0f4f482512891d20a145efa89042e 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -609,7 +609,11 @@ async function zip(config) {
 }
 async function zipDir(directory, outputPath, options) {
   const archive = new JSZip();
-  const files = (await glob2("**/*", {
+  const files = (await glob2([
+    "**/*",
+    /* workaround for https://github.com/wxt-dev/wxt/issues/604 */
+    ...options?.include || []
+  ], {
     cwd: directory,
     // Ignore node_modules, otherwise this glob step takes forever
     ignore: ["**/node_modules"],
